frontend/src/routes/(voters)/candidates/+page.svelte

```diff
@@ -11,14 +11,18 @@
   const urlRoot = $page.url.pathname.replace(/\/$/, '');
 </script>
 
-<div class="flex h-full flex-col items-center justify-center bg-primary pb-12">
-  <div class="max-w-xl">
-    <div class="p-5">
-      <h1 class="ml-2.5 mt-14 text-center text-3xl font-medium leading-6 text-gray-500">
-        {$_('candidates.candidates')}
-      </h1>
-    </div>
-    <div role="feed" class="grid grid-cols-1 gap-4" aria-label={$_('candidates.candidates')}>
+<div
+  class="flex w-full flex-grow flex-col items-center justify-center bg-base-300 p-lg pb-[3.5rem]">
+  <div class="w-full max-w-xl">
+    <hgroup>
```

Comment:
Why do we need to wrap the `h1` tag in an `hgroup`?

Reply:
True it's not necessary. I was just using those to emulate the layout in the upcoming `Page` update but I'll remove them from this PR.